File: C/projects/CARDAMOM_MODELS/DALEC/DALEC_ALL/HYDROLOGY_MODULES/CONVERTERS/HYDROFUN_EWT2MOI.c
#pragma once

/* 
Convert equivalent water thickness (mm or kg/m2) to volumetric soil moisture (m3/m3)
Inputs:
ewt - equivalent water thickness (mm)
p - layer porosity (1)
z - layer thickness (m)
*/

double HYDROFUN_EWT2MOI(double ewt, double p, double z){

double moi=ewt/(1000*p*z);

return moi;
}

File: PYTHON/projects/CARDAMOM_MODELS/DALEC/DALEC_ALL/HYDROLOGY_MODULES/CONVERTERS/HYDROFUN_EWT2MOI.py
import jax.numpy as jnp

def HYDROFUN_EWT2MOI(ewt, p, z):
    """Convert equivalent water thickness (mm or kg/m2) to volumetric soil moisture (m3/m3)"""
    return ewt / (1000.0 * p * z + 1e-9)
